api/src/main/java/org/apache/cloudstack/api/command/admin/usage/ListUsageRecordsCmd.java

```diff
@@ -85,6 +85,10 @@
     @Parameter(name = ApiConstants.OLD_FORMAT, type = CommandType.BOOLEAN, description = "Flag to enable description rendered in old format which uses internal database IDs instead of UUIDs. False by default.")
     private Boolean oldFormat;
 
+    @Parameter(name = ApiConstants.IS_RECURSIVE, type = CommandType.BOOLEAN,
+            description = "Specify if usage records should be fetched recursively per domain.")
```

Comment:
@Spaceman1984 you may add the `since` key here

Reply:
Done